mne/io/eeglab/eeglab.py

```diff
@@ -298,7 +310,10 @@ def __init__(self, input_fname, montage, eog=(), preload=False,
                             ' the .set file contains epochs.' % eeg.trials)
 
         last_samps = [eeg.pnts - 1]
-        info = _get_info(eeg, montage, eog=eog)
+        info, eeg_montage, update_ch_names = _get_info(eeg, montage, eog=eog)
```

Comment:
So this PR does not accomplish the goal, which is to make it that `montage` is only used at the end of `__init__`?

Reply:
it does for `RawEEGLAB`, not for `EpochEEGLAB` this is a different beast doesn't have a set_montage.

Reply:
are you referring to the fact that `_get_info` requires `montage`? I need it just to know if it's None to drive the calibration set. But is not used. I could change it for `is_montage_none` (or similar) to pass instead of passing `montage`. But I thought it was not worth it. 

the calibration in info could be tweaked outside of `_get_info` based on montage, but it was less readable.

Reply:
I would rather let this part with montage, then make a second PR that changes these lines of code and fixes/deprecate whatever gets broken.